src/otx/algo/detection/heads/rtmdet_head.py

```diff
@@ -152,11 +166,28 @@ def loss_by_feat_single(  # type: ignore[override]
         bbox_targets: Tensor,
         assign_metrics: Tensor,
         stride: list[int],
-    ) -> tuple[torch.Tensor, ...]:
-        """Compute loss of a single scale level."""
-        if stride[0] != stride[1]:
-            msg = "The stride of the feature map should be the same."
-            raise ValueError(msg)
+    ) -> tuple[Tensor, ...]:
+        """Compute loss of a single scale level.
+
+        Args:
+            cls_score (Tensor): Box scores for each scale level
+                Has shape (N, num_anchors * num_classes, H, W).
+            bbox_pred (Tensor): Decoded bboxes for each scale
+                level with shape (N, num_anchors * 4, H, W).
+            labels (Tensor): Labels of each anchors with shape
+                (N, num_total_anchors).
+            label_weights (Tensor): Label weights of each anchor with shape
+                (N, num_total_anchors).
+            bbox_targets (Tensor): BBox regression targets of each anchor with
+                shape (N, num_total_anchors, 4).
+            assign_metrics (Tensor): Assign metrics with shape
+                (N, num_total_anchors).
+            stride (list[int]): Downsample stride of the feature map.
+
+        Returns:
+            dict[str, Tensor]: A dictionary of loss components.
+        """
+        assert stride[0] == stride[1], "h stride is not equal to w stride!"  # noqa: S101
```

Comment:
Same as above

Reply:
No reason. I got mmdeploy source as it is and ignored ruff error.

Reply:
Done at https://github.com/openvinotoolkit/training_extensions/pull/3542/commits/d7a4e92ff4ddef6b226fa00543fa24fbc8233cfd.